### DISCUSSION TITLE
Tixati

### DISCUSSION THREAD

**Discussion by theolodger** [catppuccin/catppuccin#1830] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Tixati does use os theme for much of the ui so not much of it can be themed
![image](https://user-images.githubusercontent.com/54070635/205068104-056be4c4-fbf3-4c29-9a1d-9fd1488d7a8f.png)

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
